Formula/chezmoi.rb

```diff
@@ -24,7 +24,7 @@ def install
       -s -w
       -X main.version=#{version}
       -X main.commit=#{Utils.git_head}
-      -X main.date=#{time.rfc3339}
+      -X main.date=#{time.strftime("%FT%T#{time.utc? ? 'Z' : '%:z'}")}
```

Comment:
`time.utc?` should always be `true`. https://rubydoc.brew.sh/Formula.html#time-instance_method